src/main/java/org/apache/ibatis/annotations/DeleteProvider.java

```diff
@@ -31,4 +31,6 @@
   Class<?> type();
 
   String method();
+
+  boolean cacheSqlSource() default false;
```

Comment:
Please add API documentation(JavaDoc).
I think that we should explain an effect and caution for this attribute.

Reply:
This attribute is used to configure whether to cache SqlSource  and avoid generating SqlSource  repeatedly. is this name appropriate?